pypowsybl/network/impl/network.py

```diff
@@ -250,32 +251,39 @@ def get_single_line_diagram(self, container_id: str, parameters: SldParameters =
     def write_network_area_diagram_svg(self, svg_file: PathOrStr, voltage_level_ids: Union[str, List[str]] = None,
                                        depth: int = 0, high_nominal_voltage_bound: float = -1,
                                        low_nominal_voltage_bound: float = -1,
-                                       edge_name_displayed: bool = False) -> None:
+                                       nad_parameters: NadParameters = None) -> None:
```

Comment:
I think that we should not break compatibility and keep the old signature as well (with a deprecated mention)

Reply:
I have a problem : the new method can not have the same name if we keep compatibility